docs/README.rst

```diff
@@ -13,7 +13,7 @@ Setup the environment
 To compile the documentation you need Sphinx Python library. To install it
```

Comment:
This file is not part of the rendered documentation, so there is not much benefit to specifying a highlighting for it. And, in any case, since all code there is Bash, it would make more sense to change the default language of the document itself using the [`highlight` directive](https://www.sphinx-doc.org/es/1.5/markup/code.html#directive-highlight).